src/pages/Compare/index.spec.tsx

```diff
@@ -1172,6 +1237,41 @@ describe(__filename, () => {
       });
     });
 
+    it('dispatches fetchDiff and fetchVersionFile for the next file when the current file was deleted', () => {
+      const addonId = nextUniqueId();
+      const baseVersionId = nextUniqueId();
+      const headVersionId = baseVersionId + 1;
+      const currentFile = 'current.json';
+      const nextModifiedFile = 'modified.json';
+      const {
+        _fetchDiff,
+        _fetchVersionFile,
+        dispatchSpy,
+        fakeThunk,
+      } = setUpForPreloadAndRender({
+        addonId,
+        baseVersionId,
+        currentFile,
+        currentFileStatus: 'D',
+        loadVersionFile: false,
+        headVersionId,
+        nextModifiedFile,
```

Comment:
Just to be explicit in this test, you should probably pass `nextModifiedFileStatus: 'M'`, even though it is the default.